src/entity/Living.php

```diff
@@ -606,6 +606,12 @@ protected function entityBaseTick(int $tickDiff = 1) : bool{
 			if($this->doAirSupplyTick($tickDiff)){
 				$hasUpdate = true;
 			}
+
+			foreach($this->armorInventory->getContents() as $item){
+				if($item->onTickWorn($this)){
```

Comment:
It's possible `onTickWorn()` may alter the item itself, e.g. elytra applying damage. In this case, the item would need to be set back into the inventory afterwards.